.github/workflows/ruff.yaml

```diff
@@ -0,0 +1,15 @@
+name: ruff
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Python Format/Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup checkout
+        uses: actions/checkout@master
+      - name: Install uv
+        uses: astral-sh/setup-uv@main
+      - name: Lint
+        run: uv run --extra dev ruff check funlib/persistence tests
```

Comment:
I think you probably want a format command here too (to replace black)

Reply:
also just want it to check the formatting, not actually change anything